app/src/main/java/com/everest/movieapp/api/Api.kt

```diff
@@ -0,0 +1,14 @@
+package com.everest.movieapp.api
+
+import com.everest.movieapp.model.MovieDb
+import retrofit2.Call
+import retrofit2.http.GET
+
+interface Api {
+
+    @GET("3/movie/popular?api_key=8ed49a9afc3f95499f3e4aed8eed5a33")
```

Comment:
Normally API key should not be added like this
We can add `interceptor` to OkHTTPClient`. we can add api_key as query param there.